packages/formation-react/src/components/modal/Modal.jsx

```diff
@@ -42,7 +42,6 @@ class Modal extends React.Component {
   teardownModal() {
     if (this.state.lastFocus) {
       this.state.lastFocus.focus();
-      this.setState({ lastFocus: null });
```

Comment:
Are there any drawbacks to not setting `lastFocus` to `null` from this line: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/201/files#diff-7018d1e165f9f39d7cddc546e8331f62R22

Reply:
@kelsonic No I don't think so because this only gets called if the next props is not visible.  That should only happen as the modal is exited and any props passed in come from vets website / redux.  If the modal is opened again, it should hit the constructor and be set to null on mount.